client/my-sites/plugins/plugin-management-v2/plugin-card/index.tsx

```diff
@@ -1,78 +1,113 @@
 import { Gridicon, Card, Button } from '@automattic/components';
-import { useTranslate } from 'i18n-calypso';
-import PluginRowFormatter from '../plugin-row-formatter';
-import UpdatePlugin from '../update-plugin';
-import type { Plugin } from '../types';
+import classNames from 'classnames';
+import type { PluginColumns, PluginRowFormatterArgs } from '../types';
 import type { SiteData } from 'calypso/state/ui/selectors/site-data';
-import type { ReactElement } from 'react';
+import type { ReactElement, ReactNode } from 'react';
 
 import './style.scss';
 
 interface Props {
-	item: Plugin;
+	item: any;
 	selectedSite: SiteData;
+	rowFormatter: ( args: PluginRowFormatterArgs ) => ReactNode;
+	columns: PluginColumns;
+	hasMoreActions: boolean;
 }
 
-export default function PluginCard( { item, selectedSite }: Props ): ReactElement {
-	const translate = useTranslate();
+export default function PluginCard( {
+	item,
+	selectedSite,
+	rowFormatter,
+	columns,
+	hasMoreActions,
+}: Props ): ReactElement {
+	const columnKeys: { [ key: string ]: boolean } = columns.reduce(
+		( obj, cur ) => ( { ...obj, [ cur.key ]: true } ),
+		{}
+	);
+
+	const showLeftContent = columnKeys[ 'plugin' ];
```

Comment:
When we don't want to display this left content HTML? Is that something related to the plugin details UI?

Reply:
Yes, something which will be needed.